__tests__/components/editor/property/InputLiteral.test.js

```diff
@@ -40,45 +46,31 @@ describe('<InputLiteral />', () => {
   })
 
   it('contains required="true" attribute on input tag when mandatory is true', () => {
-    wrapper.instance().props.propertyTemplate.mandatory = 'true'
-    wrapper.instance().forceUpdate() /** Update plProps with mandatory: "true" * */
+    const propertyTemplate = { propertyTemplate: { ...plProps.propertyTemplate, mandatory: 'true' } }
+    wrapper.setProps({ ...plProps, ...propertyTemplate })
     expect(wrapper.find('input').prop('required')).toBeTruthy()
     expect(wrapper.find('label > RequiredSuperscript')).toBeTruthy()
   })
 
   it('contains required="false" attribute on input tag when mandatory is false', () => {
-    wrapper.instance().props.propertyTemplate.mandatory = 'false'
-    wrapper.instance().forceUpdate()
+    const propertyTemplate = { propertyTemplate: { ...plProps.propertyTemplate, mandatory: 'false' } }
+    wrapper.setProps({ ...plProps, ...propertyTemplate })
     expect(wrapper.find('input').prop('required')).toBeFalsy()
   })
 })
 
-describe('checkMandatoryRepeatable', () => {
```

Comment:
why don't we want / need this test any more?  I suspect we will see all the variants of mandatory true/false and repeatable true/false ...

Is this duplicated elsewhere?

Reply:
There is no such thing as "checkMandatoryRepeatable." I think this is a relic from another era. We are already testing `mandatory` on 48-59

Reply:
I was interpreting the test as "check when both mandatory AND repeatable are true" but whatever.